File: Sources/CLI/Models/Account.swift
//
//  Account.swift
//  IPATool
//
//  Created by Majd Alfhaily on 21.03.22.
//

import Foundation

struct Account: Codable {
    let name: String
    let email: String
    let passwordToken: String
    let directoryServicesIdentifier: String
}

File: pkg/appstore/account.go
package appstore

type Account struct {
	Email               string `json:"email,omitempty"`
	PasswordToken       string `json:"passwordToken,omitempty"`
	DirectoryServicesID string `json:"directoryServicesIdentifier,omitempty"`
	Name                string `json:"name,omitempty"`
	StoreFront          string `json:"storeFront,omitempty"`
	Password            string `json:"password,omitempty"`
}
